kyo-data/shared/src/main/scala/kyo/Text.scala

```diff
@@ -0,0 +1,288 @@
+package kyo
+
+import Text.internal.*
+import scala.annotation.tailrec
+
+opaque type Text >: String = String | Op
+
+object Text:
+
+    def apply(s: String): Text = s
```

Comment:
Just a helper in cases where inference doesn't transform a `String` into `Text`, similarly to `Kyo.pure`.